packages/exo/src/exo-tools.js

```diff
@@ -67,16 +159,15 @@ const defendSyncArgs = (syncArgs, methodGuardPayload, label = undefined) => {
  */
 const defendSyncMethod = (method, methodGuardPayload, label) => {
   const { returnGuard } = methodGuardPayload;
+  const isRawReturn = isRawValueGuard(returnGuard);
+  const redactConfig = redactRawValues(methodGuardPayload);
```

Comment:
Since you get the `returnGuard` from the `methodGuardPayload` anyway, and since you call the value it returns `redactConfig`, does it read slightly better to move line 162 into `redactRawValues` and include `isRawReturn` as a property of the `redactConfig`? (If so, then for my type name suggestions above, only consider `RedactConfig`.)

Reply:
Oh, I see. Not as much nicer as I thought, because the return value processing must still be separate here. Nevermind. (But left original comment as record)